serving/samples/helloworld-vertx/README.md

```diff
@@ -0,0 +1,224 @@
+            <version>${version.vertx}</version>
+        </dependency>
+    </dependencies>
+
+    <build>
+        <plugins>
+            <plugin>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <version>3.8.0</version>
+                <configuration>
+                    <source>1.8</source>
+                    <target>1.8</target>
+                </configuration>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-shade-plugin</artifactId>
+                <version>3.2.0</version>
+                <executions>
+                    <execution>
+                        <phase>package</phase>
+                        <goals>
+                            <goal>shade</goal>
+                        </goals>
+                        <configuration>
+                            <transformers>
+                                <transformer
+                                        implementation="org.apache.maven.plugins.shade.resource.ManifestResourceTransformer">
+                                    <manifestEntries>
+                                        <Main-Class>io.vertx.core.Launcher</Main-Class>
+                                        <Main-Verticle>com.example.helloworld.HelloWorld</Main-Verticle>
+                                    </manifestEntries>
+                                </transformer>
+                            </transformers>
+                            <artifactSet/>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+
+        </plugins>
+    </build>
+    <properties>
+        <version.vertx>3.5.4</version.vertx>
+    </properties>
+    </project>
+    ```
+
+1. Create a `src/main/java/com/example/helloworld` folder, then create a new file named `HelloWorld.java` in that folder
+   and paste the following code. This code creates a basic web server which
+   listens on port 8080:
+
+   ```java
+    package com.example.helloworld;
+
+    import io.reactivex.Flowable;
+    import io.vertx.reactivex.core.AbstractVerticle;
+    import io.vertx.reactivex.core.http.HttpServer;
+    import io.vertx.reactivex.core.http.HttpServerRequest;
+
+    public class HelloWorld extends AbstractVerticle {
+
+        public void start() {
+
+            final HttpServer server = vertx.createHttpServer();
+            final Flowable<HttpServerRequest> requestFlowable = server.requestStream().toFlowable();
+
+            requestFlowable.subscribe(httpServerRequest -> {
+
+                String target = System.getenv("TARGET");
+                if (target == null) {
+                    target = "NOT SPECIFIED";
+                }
+
+                httpServerRequest.response().setChunked(true)
+                        .putHeader("content-type", "text/plain")
+                        .setStatusCode(201) // accepted
```

Comment:
the actual sample Java file included in this same PR responds with `200` (and that's probably more appropriate in this case?)